application/src/main/java/uk/nhs/hee/tis/revalidation/service/RecommendationServiceImpl.java

```diff
@@ -347,32 +340,6 @@ public RecommendationStatus getRecommendationStatusForTrainee(String gmcId) {
     return currentRecommendations;
   }
 
-  /**
-   * Check all of the incomplete recommendations for a Doctor and if GMC update status with Approved
-   * or Rejected, recommendation will be moved to snapshot
-   *
-   * @param gmcNumber      The GMC ID/Number of the doctor to check recommendations for
-   * @param designatedBody The Body responsible for the recommendations/revalidation
-   */
-  private void checkRecommendationStatus(final String gmcNumber, final String designatedBody) {
```

Comment:
I think "checkRecommendationStatus" method's logic is now part of GmcClientService class?

Reply:
GmcClientService got its own public method `checkRecommendationStatus`. The private method I remove there is just for internal use in RecommendationServiceImpl.